src/uct/ib/dc/dc_mlx5.h

```diff
@@ -99,9 +99,16 @@ typedef enum {
 
 typedef struct uct_dc_dci {
     uct_rc_txqp_t                 txqp; /* DCI qp */
-    uct_dc_mlx5_ep_t              *ep;  /* points to an endpoint that currently
+    union {
+        uct_dc_mlx5_ep_t          *ep;  /* points to an endpoint that currently
```

Comment:
there are multiple places which access dci->ep, are we sure they all check the policy is dcs before doing that?